layers/exporters/to_svg.py

```diff
@@ -8,8 +10,278 @@
+            :param font: What font style to use - "sans", "sans-serif", or "monospace"
+            :param tableBorderColor: Hex color to use for the technique borders
+            :param showHeader: Whether or not to show Header Blocks
+            :param legendDocked: Whether or not the legend should be docked
+            :param legendX: Where to place the legend on the x axis if not docked
+            :param legendY: Where to place the legend on the y axis if not docked
+            :param legendWidth: Width of the legend if not docked
+            :param legendHeight: Height of the legend if not docked
+            :param showLegend: Whether or not to show the legend
+            :param showFilters: Whether or not to show the Filter Header Block
+            :param showAbout: Whether or not to show the About Header Block
+            :param border: What default border width to use
+        """
+        self.width = width
+        self.height = height
+        self.headerHeight = headerHeight
+        self.unit = unit
+        self.showSubtechniques = showSubtechniques
+        self.font = font
+        self.tableBorderColor = tableBorderColor
+        self.showHeader = showHeader
+        self.legendDocked = legendDocked
+        self.legendX = legendX
+        self.legendY = legendY
+        self.legendWidth = legendWidth
+        self.legendHeight = legendHeight
+        self.showLegend = showLegend
+        self.showFilters = showFilters
+        self.showAbout = showAbout
+        self.border = border
+
+    def load_from_file(self, filename=""):
+        """
+            Load config from a json file
+
+            :param filename: The file to read from
+        """
+        with open(filename, 'r') as fio:
+            raw = fio.read()
+            self._data = json.loads(raw)
+        for entry in self._data:
+            if entry in vars(self).keys():
+                setattr(self, entry, self._data[entry])
+            else:
+                print('WARNING - Unidentified Config Field in {}: {}'.format(filename, entry))
+
+        self._display()
+
+    def save_to_file(self, filename=""):
+        """
+            Store config to json file
+
+            :param filename: The file to write to
+        """
+        out = dict(width=self.width, height=self.height, headerHeight=self.headerHeight, unit=self.unit,
+                   showSubtechniques=self.showSubtechniques, font=self.font, tableBorderColor=self.tableBorderColor,
+                   showHeader=self.showHeader, legendDocked=self.legendDocked, legendX=self.legendX,
+                   legendY=self.legendY, legendWidth=self.legendWidth, legendHeight=self.legendHeight,
+                   showLegend=self.showLegend, showFilters=self.showFilters, showAbout=self.showAbout,
+                   border=self.border)
+        with open(filename, 'w') as file:
+            json.dump(out, file)
+
+    def _display(self):
```

Comment:
It could be. I borrowed the convention from another python project I'm on that's tried to avoid using `__str__()`, but I don't see any real reason to avoid it here.